build.gradle.kts

```diff
@@ -34,6 +34,11 @@ object Versions {
 configurations {
     register("testRuntimeDependencies") {
         extendsFrom(compileOnly.get())
+        attributes {
+            // KGP publishes multiple variants https://kotlinlang.org/docs/whatsnew17.html#support-for-gradle-plugin-variants
+            attribute(Usage.USAGE_ATTRIBUTE, project.objects.named(Usage.JAVA_RUNTIME))
+            attribute(Category.CATEGORY_ATTRIBUTE, project.objects.named(Category.LIBRARY))
+        }
```

Comment:
This turned out to be pretty neat feature, more info: https://youtrack.jetbrains.com/issue/KT-52966